.rubocop.yml

```diff
@@ -51,7 +51,7 @@ Metrics/BlockLength:
   Exclude:
     - rubocop-rspec.gemspec
     - Rakefile
-    - '**/*.rake'
+    - "**/*.rake"
```

Comment:
I am not sure about this line, but the other changes look fine.

Reply:
Oops, looks like prettier has made that change. I reverted it back.